.changeset/eleven-women-glow.md

```diff
@@ -0,0 +1,7 @@
+---
```

Comment:
I think you should mention, either in this change set or a second one, that users of the alpha APIs can now put metadata on AllowedTypes.